### PR TITLE
Drop remaining mentions of evict extension point

### PR DESCRIPTION
**Pull request by ingvagabund**
Follow up for https://github.com/kubernetes-sigs/descheduler/pull/1120.

### CODE REVIEW

File: pkg/descheduler/policyconfig_test.go [hunk 1]
```diff
@@ -878,9 +878,6 @@ profiles:
       filter:
         enabled:
           - "DefaultEvictor"
-      evict:
-        enabled:
-          - "DefaultEvictor"
```
Comment (damemi) [inline pkg/descheduler/policyconfig_test.go:883]: no `preEvictionFilter` here? Or is that already tested?

**Review by damemi — COMMENTED** [review]
/approve
though if DefaultEvictor is enabled by default when unset in filter/preEvictionFilter, we can probably just exclude it from the readme+examples. 

One section on how to configure it and explaining that it's on by default is better than adding it to every other example imo. what do you think?

**Comment by ingvagabund** [comment]
@damemi I wanted to be more explicit. However, you are right there's no need to mention it everywhere. I have update the generic description of the policy example. We still don't provide a way to disable any extension point. That will be implement in the next release once the framework gets completed.

File: pkg/descheduler/policyconfig_test.go (hunk 1 above)
Comment (ingvagabund) [reply pkg/descheduler/policyconfig_test.go:883]: Dropped the `filter` ep as well. `TestDecodeDefaults` provides testing for the missing eps.

**Comment by knelasevero** [comment]
If we show yamls that are not complete, people will apply those and complain. Unless we explain to them that they are not complete

**Comment by knelasevero** [comment]
I still prefer to leave all config examples with a default evictor included. Also for copy paste value.

**Comment by damemi** [comment]
> If we show yamls that are not complete, people will apply those and complain. Unless we explain to them that they are not complete

As I understood it, these yamls will work even without including the DefaultEvictor since it's enabled by default: https://github.com/kubernetes-sigs/descheduler/blob/a7113dab5048c556046fb61addd2ce84f9760094/pkg/descheduler/policyconfig.go#L110-L117

So it should be copy-pastable either way. But I'm fine following up with this and updating the docs later

**Comment by knelasevero** [comment]
that's true

don't know why I forgot about this. I guess because in all my local yamls I add a default evictor with muscle memory :sweat: 

Disregard my previous comment then

**Pull request merged**